edit.go

```diff
@@ -422,6 +433,18 @@ func printSSHKeys(v *vaulted.Vault) {
 	}
 }
 
+func printDuration(v *vaulted.Vault) {
+	cyan := color.New(color.FgCyan)
+	cyan.Printf("\nEnvironment Duration (in minutes): ")
+	var duration int64
+	if v.Duration == 0 {
+		duration = vaulted.STS_DURATION_DEFAULT
+	} else {
+		duration = v.Duration
+	}
+	fmt.Printf("%d\n", int64(duration)/60)
```

Comment:
If the `duration` were already a `time.Duration`, you could simply call `duration.String()` (or format it with `"%v"`).